buildscripts/smoke/smoke.go

```diff
@@ -0,0 +1,195 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"math/rand"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	port = 8080
```

Comment:
This may be paranoid but it would be safer to pass the port in as a command line argument when running this and set the port explicitly in the makefile. Not a big deal though, feel free to ignore.

Reply:
I opted to pass the port as an environment variable.